packages/runtime-core/src/components/KeepAlive.ts

```diff
@@ -127,6 +127,8 @@ const KeepAliveImpl: ComponentOptions = {
 
     sharedContext.activate = (vnode, container, anchor, isSVG, optimized) => {
       const instance = vnode.component!
+      // on activation, resume the effect of the component instance and immediately execute the call during the pause process
+      instance.effect.resume(true)
```

Comment:
I wonder if we could have a prop for KeepAlive to toggle this behavior (and default disabling it to keep existing behaviour)

Reply:
Yes, we can add a prop to preserve the original behavior.

Reply:
I added a `lazy` prop to control whether it should pause in the deactivated state.